dist/datatypes/minecraft.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
```

Comment:
dist shouldn't be commited

Reply:
Yeah there’s a lot of leftover code and dependencies that need to be removed. I think filfat was using Babel, it’s not required anymore